packages/server-core/src/recording/recording/recording.class.ts

```diff
@@ -23,99 +23,90 @@ All portions of the code written by the Ethereal Engine team are Copyright © 20
 Ethereal Engine. All Rights Reserved.
 */
 
-import { Paginated } from '@feathersjs/feathers'
-import { SequelizeServiceOptions, Service } from 'feathers-sequelize'
-
-import { RecordingResult } from '@etherealengine/common/src/interfaces/Recording'
-
-import { RecordingID } from '@etherealengine/common/src/interfaces/RecordingID'
-import { recordingResourcePath } from '@etherealengine/engine/src/schemas/recording/recording-resource.schema'
-import { UserType, userPath } from '@etherealengine/engine/src/schemas/user/user.schema'
-import { Forbidden, NotFound } from '@feathersjs/errors'
+import type { Paginated, PaginationOptions, Params } from '@feathersjs/feathers'
+import type { KnexAdapterOptions } from '@feathersjs/knex'
+import { KnexAdapter } from '@feathersjs/knex'
+
+import {
+  RecordingData,
+  RecordingID,
+  RecordingPatch,
+  RecordingQuery,
+  RecordingType
+} from '@etherealengine/engine/src/schemas/recording/recording.schema'
+import { NotFound } from '@feathersjs/errors'
 import { Application } from '../../../declarations'
-import { UserParams } from '../../api/root-params'
+import { RootParams } from '../../api/root-params'
 import { checkScope } from '../../hooks/verify-scope'
 
-export type RecordingDataType = RecordingResult
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface RecordingParams extends RootParams<RecordingQuery> {}
 
-export class Recording<T = RecordingDataType> extends Service<T> {
+export class RecordingService<T = RecordingType, ServiceParams extends Params = RecordingParams> extends KnexAdapter<
+  RecordingType,
+  RecordingData,
+  RecordingParams,
+  RecordingPatch
+> {
   app: Application
-  docs: any
-  constructor(options: Partial<SequelizeServiceOptions>, app: Application) {
+
+  constructor(options: KnexAdapterOptions, app: Application) {
     super(options)
     this.app = app
   }
 
-  async get(id: RecordingID, params?: any): Promise<T> {
-    // get resources with associated URLs
-    // TODO: move resources population to resolvers once this service is migrated to feathers 5
-    const resources = await this.app.service(recordingResourcePath).find({
-      query: {
-        recordingId: id
-      },
-      paginate: false
-    })
-
-    const result = (await super.get(id)) as RecordingDataType
-
-    result.resources = resources.map((resource) => resource.staticResource)
-
-    return result as T
+  async get(id: RecordingID, params?: RecordingParams) {
+    return await super._get(id, params)
   }
 
-  async find(params?: UserParams): Promise<Paginated<T>> {
+  async find(
+    params?: RecordingParams & {
+      paginate?: PaginationOptions | false
```

Comment:
We no longer need this `paginate` to be appended here as RootParams already have it.